app/inputs/multi_value_ordered_input.rb

```diff
@@ -59,6 +59,31 @@ def build_field(value, index)
       index = value.object_id
     end
 
+    if value.is_a?(NestedOrderedCreator)
+      item_options = build_item_options(value, index, :creator)
+      input_creator = @builder.text_field(:creator, item_options)
+    elsif value.is_a?(NestedOrderedTitle)
+      item_options = build_item_options(value, index, :title)
+      input_creator = @builder.text_field(:title, item_options)
```

Comment:
I think it would be a good idea to rename these variables to match the nested field names for consistency, so the line above coule be: 
```
input_title = @builder.text_field(:title, item_options)
```

I did some of of these changes in a previous PR, see https://github.com/osulp/Scholars-Archive/pull/1628/files#diff-057c8e520a5ab3d69344bf096f4e5553R69

I basically moved the logic to a new method `build_nested_item(value,index)` and renamed the variables there. This method returns the appropriate inputs based on the nested models.